Src/Support/Google.Apis.Core/Http/ConfigurableMessageHandler.cs

```diff
@@ -609,6 +637,38 @@ bool DisposeAndReturnFalse(IDisposable disposable)
             return response;
         }
 
+        private async Task CredentialInterceptAsync(HttpRequestMessage request, CancellationToken cancellationToken)
+        {
+            var actualCredential = GetActualCredential(request);
+            if (actualCredential != null)
+            {
+                await actualCredential.InterceptAsync(request, cancellationToken).ConfigureAwait(false);
+            }
+        }
+
+        private async Task<bool> CredentialHandleResponseAsync(HandleUnsuccessfulResponseArgs args)
+        {
+            var actualCredential = GetActualCredential(args.Request);
+            if (actualCredential != null && actualCredential is IHttpUnsuccessfulResponseHandler handler)
```

Comment:
I don't think you need the != null check here. It can't be an IHttpUnsuccessfulResponseHandler if it's null.